.gitattributes

```diff
@@ -0,0 +1,23 @@
+# GitHub Linguist overrides
+
+# Directory-level overrides
+doc/** linguist-documentation
+
+# Connecting languages to explicit file types
+*.C linguist-language=C++
+*.h linguist-language=C++
```

Comment:
This line is different from the MOOSE version, which does not reference the header files (could be my local MOOSE is also out of date). Including header files here seems correct, yet would confirm for me that the inclusion is intentional here?

Reply:
This was more for completeness/consistency. I think if you click on "C++" on MOOSE, then it'll just show things in `src`. I always thought it odd that ".h" wasn't included, because it's tracked in the linguist application that github uses for this.